pyang/plugins/flatten.py

```diff
@@ -235,10 +235,21 @@ def output_module(
         override_flag=None,
         known_keys=None,
     ):
-        module_children = (
+        module_children = [
             child
             for child in getattr(module, "i_children", [])
             if child.keyword in self.__keywords
+        ]
+        if ctx.opts.flatten_deviated:
+            deviated_module_children = [
+                child
+                for child in getattr(module, "i_not_supported", [])
+                if child.keyword in self.__keywords
+            ]
+            module_children = module_children + deviated_module_children
+        module_children = sorted(
+            module_children,
+            key=lambda child: statements.get_xpath(child, prefix_to_module=(not ctx.opts.flatten_prefix_in_xpath), qualified=ctx.opts.flatten_qualified_in_xpath)
```

Comment:
Please make this line to multi lines in order to be shorter than 80 chars

Reply:
Done.